server/config/passport.js

```diff
@@ -43,7 +43,7 @@ passport.use(new LocalStrategy({ usernameField: 'email' }, (email, password, don
  * Authentificate using Basic Auth (Username + Api Key)
  */
 passport.use(new BasicStrategy((userid, key, done) => {
-  User.findOne({ username: userid }, (err, user) => { // eslint-disable-line consistent-return
+  User.findOne({ username: userid }).collation({ locale: 'en', strength: 2 }).exec((err, user) => { // eslint-disable-line consistent-return
```

Comment:
We could add statics on the `User` model called `User.findByUsername()` and `User.findByEmail()` that will perform the `User.findOne().collation(...)` query.

This avoids duplication and ensures we won't forget to call it. It also means those statics can have a comment about what `collation` is doing as it's not very obvious.

Reply:
Thanks for the suggestion! Good idea.